ingest/rules/curate.smk

```diff
@@ -116,6 +116,23 @@ rule curate:
         """
 
 
+rule add_genbank_url:
+    input:
+        metadata=temp("data/all_metadata_intermediate.tsv"),
+    output:
+        metadata="data/all_metadata.tsv",
+    log:
+        "logs/add_genbank_url",
+    benchmark:
+        "benchmarks/add_genbank_url",
+    shell:
+        r"""
+        csvtk mutate2 -tl \
```

Comment:
Based on [TSV docs](https://docs.nextstrain.org/en/latest/reference/data-formats.html#tsv), the `-l` flag is not necessary 

```suggestion
        csvtk mutate2 -t \
```

Reply:
[Stolen from Zika](https://github.com/nextstrain/zika/blob/649e495a6b181c004bacae26ea4507fc6989abc5/phylogenetic/rules/merge_sequences_usvi.smk#L44), and [also found in mpox, measles, dengue, and rsv](https://github.com/search?q=org%3Anextstrain+-tl&type=code) — but I'll remove it here.

Reply:
Removing the `-l` flag causes the command to fail with this error:

> [ERRO] parse error on line 639, column 230: bare " in non-quoted-field

so I'm gonna leave the `-l` in, I guess.

Reply:
Ah, this requires [Augur v26.2.0](https://github.com/nextstrain/augur/releases/tag/26.2.0), which includes the change for `augur curate` to output CSV-like TSV. I don't see the error after running `nextstrain update docker`.

Reply:
Ope, yep, confirmed that all is well after an update. (I really should throw that in a cron job or something…) `-l`-less version force-pushed.